extern/nativefiledialog/src/simple_exec.h

```diff
@@ -1,4 +1,5 @@
 // copied from: https://github.com/wheybags/simple_exec/blob/5a74c507c4ce1b2bb166177ead4cca7cfa23cb35/simple_exec.h
+// modified #ifndef SIMPLE_EXEC_H header guard to #pragma once
```

Comment:
This is a mistake?

Reply:
Yes. This slipped through during my decision to not touch extern.

Reply:
Done